examples/pytorch/ogb/directional_GSN/preprocessing.py

```diff
@@ -0,0 +1,285 @@
+from ogb.graphproppred import DglGraphPropPredDataset, Evaluator
```

Comment:
There are so many functions defined in this script. Perhaps you could merge some of them.